src/data/actions/assignmentReminder.js

```diff
@@ -0,0 +1,47 @@
+import {
+  ASSIGNMENT_REMINDER_REQUEST,
```

Comment:
nits:
1. Rename file to `codeReminder.js`
2. Rename constants to be `CODE_REMINDER_XX`